src/Microsoft.Azure.WebJobs.Extensions.Kafka/Config/KafkaExtensionConfigProvider.cs

```diff
@@ -52,12 +52,9 @@ public void Initialize(ExtensionConfigContext context)
             this.configuration.ConfigurationSection.Bind(this.options);
 
             context
-               .AddConverter<string, KafkaEventData>(ConvertString2KafkaEventData)
                .AddConverter<KafkaEventData, string>(ConvertKafkaEventData2String)
                .AddConverter<KafkaEventData, ISpecificRecord>(ConvertKafkaEventData2AvroSpecific)
-               .AddConverter<byte[], KafkaEventData>(ConvertBytes2KafkaEventData)
-               .AddConverter<KafkaEventData, byte[]>(ConvertKafkaEventData2Bytes)
-               .AddOpenConverter<OpenType.Poco, KafkaEventData>(ConvertPocoToKafkaEventData);
+               .AddConverter<KafkaEventData, byte[]>(ConvertKafkaEventData2Bytes);
```

Comment:
I don't believe this converter is valuable either. Its implementation simply takes `value.ToString()` and does a `GetBytes()` on it. In many cases, `value.ToString()` will simply be the FQN of the type. Do we have a use for this functionality?

Reply:
I think it does. If you define that the value from Kafka will be byte[] then KafkaEventData.Value will be byte[]. The converter will be called if you define the parameter type in your function to be of type byte[] as:

```csharp
[Function("asbyte")]
public static void MyFunction([KafkaTrigger] byte[] rawValue) {
}
```